libs/@hashintel/petrinaut/src/petrinaut-sdcpn/core/helpers/compute-next-frame.test.ts

```diff
@@ -22,16 +22,22 @@ describe("computeNextFrame", () => {
           ],
         },
       ],
-      differentialEquations: [],
+      differentialEquations: [
+        {
+          id: "diffeq1",
+          name: "Differential Equation 1",
+          typeId: "type1",
+          code: "export default Dynamics((tokens, parameters) => { return tokens.map(token => ({ x: 1, y: 1 })); });",
+        },
+      ],
       parameters: [],
       places: [
         {
           id: "p1",
-          name: "Place 1",
+          name: "Place1",
           type: "type1",
           dynamicsEnabled: true,
-          differentialEquationCode:
-            "export default Dynamics((placeValues, t, parameters) => { return new Float64Array([1, 1]); });",
+          differentialEquationCode: { refId: "diffeq1" },
```

Comment:
Appreciate the field already exists, but can we rename this to `diffEqId` or something if that's what we mean? Calling things "ref" is confusing in a React context if there's no particular reason to. 

cc @kube in case this will create horrible merge conflicts. can always do in a follow-up